bin/snap-sync

```diff
@@ -191,13 +197,23 @@ fi
 
 if [[ "$($ssh findmnt -n -v --target / -o FSTYPE)" == "btrfs" ]]; then
     EXCLUDE_UUID=$($ssh findmnt -n -v -t btrfs --target / -o UUID)
-    TARGETS=$($ssh findmnt -n -v -t btrfs -o UUID,TARGET --list | grep -v $EXCLUDE_UUID | awk '{print $2}')
+    DETECTED_TARGETS=$($ssh findmnt -n -v -t btrfs -o UUID,TARGET --list | grep -v $EXCLUDE_UUID | awk '{print $2}')
     UUIDS=$($ssh findmnt -n -v -t btrfs -o UUID,TARGET --list | grep -v $EXCLUDE_UUID | awk '{print $1}')
 else
-    TARGETS=$($ssh findmnt -n -v -t btrfs -o TARGET --list)
+    DETECTED_TARGETS=$($ssh findmnt -n -v -t btrfs -o TARGET --list)
     UUIDS=$($ssh findmnt -n -v -t btrfs -o UUID --list)
 fi
 
+if [[ "$mountpoint_cmdline" == "none" ]]; then
+    TARGETS=$DETECTED_TARGETS
+    else
+        if grep -q "$mountpoint_cmdline" <<< "$DETECTED_TARGETS" ;then
+            TARGETS=$mountpoint_cmdline
```

Comment:
Wouldnt it make sense to escape the variable here when the path contains spaces?